.github/workflows/ci.yml

```diff
@@ -93,20 +99,41 @@ jobs:
             runner: "ubuntu-22.04"
     steps:
       - uses: actions/checkout@v4
-        if: github.event_name == 'merge_group'
       - name: Install Nix on ${{ matrix.systems.nix-system }} system
-        if: github.event_name == 'merge_group'
         uses: DeterminateSystems/nix-installer-action@main
         with:
-          flakehub: true
+          _internal-obliterate-actions-id-token-request-variables: ${{ matrix.id-token == 'none' }}
```

Comment:
what's this for? `_internal-obliterate...` ?

Reply:
I want those actions to not have the ACTIONS_ID_TOKEN_REQUEST_* environment variables, so they can't create an ID token and log in to flakehub. This simulates not having the `id-token: write` permission. I wanted to parameterize that permission in the matrix, but permissions can't be variable.